src/services/config.js

```diff
@@ -70,6 +71,7 @@ class ConfigData {
       } else {
```

Comment:
Should the line above be reading from `this.profiles` first? Meaning, look for the profile and API keys in `this.profiles` first and then fallback to `this.projects`/keytar.

Reply:
Thanks for pointing. Updated Accordingly.